doc/examples/returning_interaction.rb

```diff
@@ -0,0 +1,30 @@
+# Example: ReturningInteraction
+# As an example we use a ReturningInteraction to update User records in the admin workspace
+
+class UsersController < BaseController
+  ...
+    def update
+      @user = Admin::Users::Update.run_returning!(
+        { user: User.find(params[:id]) }
```

Comment:
```suggestion
        { user: User.find(params[:id]) }.merge(params[:user])
```
This is probably not exactly what we need, however, it shows the idea that we also need to pass first_name and last_name to the interaction.